controllers/release/controller.go

```diff
@@ -74,6 +76,7 @@ func (c *Controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	return controller.ReconcileHandler([]controller.Operation{
 		adapter.EnsureFinalizersAreCalled,
 		adapter.EnsureReleaseIsRunning,
+		adapter.EnsureConfigIsLoaded,
```

Comment:
This should be the first one probably. Maybe the second. In addition, if you check the internal-services repo you will see this comment:
```
// This operation sets the config in the adapter to be used in other operations.
```
I actually hate that but I think it's the cleanest. Maybe we should use different names for this operations that update the adapter... Or maybe this should not be an operation at all and happen in the newAdapter. Can we discuss it?

Reply:
comment added and order changed